README.md

```diff
@@ -38,16 +38,21 @@ Enjoy your new RPM.
 
 ### Ubuntu/Debian
 
-Building a .deb should be straight forward with:
+On recent Debian/Ubuntu distributions you should be able to install with:
+```
+# apt-get install TODO
```

Comment:
Ah okay. I thought you meant build dependencies from that line. Yes it's just as you said, either of them would work.
network-manager-ssh recommends -gnome. So just installing that one should do the trick most of the time.

You can check the status of the package on this site btw: https://packages.qa.debian.org/n/network-manager-ssh.html (https://packages.qa.debian.org/n/network-manager-ssh.html)
The package is also available on most architectures: https://buildd.debian.org/status/package.php?p=network-manager-ssh (https://buildd.debian.org/status/package.php?p=network-manager-ssh)
except for the few odd ones where the dependencies are missing right now. I mean the likelyhood that gnome would run on an
Atari is fairly low to begin with.

May 18 2016 11:57 AM, "Dan Fruehauf"  wrote:
	In README.md (https://github.com/danfruehauf/NetworkManager-ssh/pull/56#discussion_r63675847):

	> @@ -38,16 +38,21 @@ Enjoy your new RPM. > > ### Ubuntu/Debian > > -Building a .deb should be straight forward with: > +On recent Debian/Ubuntu distributions you should be able to install with: > +``` > +# apt-get install TODO

	@lhw (https://github.com/lhw) I meant more like the nm ssh packages they should apt-get install.Probably network-manager-ssh and network-manager-ssh-gnome?

	—
You are receiving this because you were mentioned.
Reply to this email directly or view it on GitHub (https://github.com/danfruehauf/NetworkManager-ssh/pull/56/files/0c5014df9a828407a4f011add5b78720f917856c#r63675847)